tests/5.1/scope/test_scope_nowait_construct.c

```diff
@@ -0,0 +1,49 @@
+//-------------test_scope_nowait_construct.c-------------//
+//
+//OpenMP API Version 5.1 Aug 2021
+//
+//Tests the behavior of the scope construct with nowait
+//specified. This test should remove the implied barrier
+//creatd by the scope construct when nowait is specified.
+//-------------------------------------------------------//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+#include <unistd.h>
+
+#define N 1024
+
+int test_scope_nowait(){
+	int errors = 0;
+	int test_arr[N];
+	for (int i=0; i<N, i++){
```

Comment:
```suggestion
	for (int i=0; i<N; i++){
```
(semicolon not colon)